### ISSUE TITLE
[Accessibility] Screen readers not compatible with context and react menus

### ISSUE THREAD

**Issue by Deakia** [Swiftgram/Telegram-iOS#196]
### Description
Apple's VoiceOver does not allow for navigation in message context menus, and message react menus or the back button in chats

### Expected Behavior
Each item should be selectable, such as Reply, Forward, Delete, Copy Link, and be able to be navigated using VoiceOver's system

### Actual Behavior
VoiceOver does not see containers for each item, only being able to select "dismiss Context Menu" and "Vertical Scroll Bar", with no other function when inside a reaction menu's popup, or message's context menu. In addition the back button in chats is not a selectable item, and the "back"gesture does not function

### Steps to Reproduce
1. Enable Apple's VoiceOver
2. Select a message, or reaction
3. Open the context menu (Swipe down twice until "open message menu" is announced
4. Attempt to return to the main chat screen by selecting the back button in the top left

### Environment
**Device:** `iPhone 15, iPad Pro 4th Gen (M2)`

**iOS version**: `26.5, 27.0`

**App version:** `12.9.2`

**Comment by Kylmakalle** [comment]
@Deakia Upstream issue, report it to bugs.telegram.org
